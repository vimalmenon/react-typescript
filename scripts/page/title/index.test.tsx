import "jsdom-global/register";
import * as React from "react";
import * as renderer from "react-test-renderer";
import Title from "./index";
import {shallow} from "enzyme";


describe("Title component :", () => {

    test("Snapshot renders :", () => {
        const component = renderer.create(<Title />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
    
    test("Title Text to match :", () => {
        const wrapper = shallow(<Title />);
        expect(wrapper.text()).toBe("COVID-19 React Tracking Application");
    });
    
});