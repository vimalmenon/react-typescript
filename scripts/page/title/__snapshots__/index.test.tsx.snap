// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component : Snapshot renders : 1`] = `
<header
  className="sc-bdnylx dpKAAl"
>
  COVID-19 React Tracking Application
</header>
`;
